2022-00065-zyxel-multicrit (#553)
DIVD-NL/web-csirt: _cases/2022/DIVD-2022-00065.md

---
layout: case
title: Multiple Critical Vulnerabilities in multiple Zyxel EOL devices
excerpt: "Based on disclosure by Sec Consult, DIVD performed scans of end of life device impacted by multiple vulnerabilities."
author: Rutger Hermens
lead: Mick Beers
researchers:
- Mick Beers
- Pepijn van der Stap
- Lennaert Oudshoorn
- Rutger Hermens
cves:

> - n/a

Reviewer: Weten we de CVE’s niet?

Author: Als ik dit document mag geloven, zijn er geen CVEs voor geregistreerd:
https://sec-consult.com/vulnerability-lab/advisory/multiple-critical-vulnerabilities-in-multiple-zyxel-devices/

Reviewer: In dat geval zou ik niet voor N/a kiezen maar een een tekst die dit omschrijft.
```suggestion
- CVE-2022-4510
- The researchers described a total of 8 vulnerabilities of which 7 have not been assigned a CVE.
```

Reviewer: Alleen deze nog?

product: 
- Zyxel
models: 
- AMG1302-T11C	EOL
- VMG3925-B10C	EOL
- VMG8924-B10D	EOL
- VMG1312-B10D	EOL
- VMG3312-T20A	EOL
- VMG3625-T20A	EOL
- VMG3925-B10B	EOL
- VMG3925-B10C	EOL
- VMG3925-B30C	EOL
- VMG3926-B10A	EOL
- VMG5313-B10B	EOL
- VMG5313-B30B	EOL
- VMG8623-T50A	EOL
- VMG8823-B10B	EOL
- VMG8823-B30B	EOL
- VMG8823-B50B	EOL
- VMG8823-B60B	EOL
- VMG8924-B10D	EOL
- VMG8924-B30D	EOL
- PMG5317-T20A	EOL

recommendation: Retire impacted devices

> #patch_status: Fully patched

Reviewer: ```suggestion
patch_status: “Devices are end-of-life and will not be patched”
```

#workaround: n/a
status : Open
start: 2022-12-19
end: 
timeline:
- start: 2022-02-15
  end:
  event: "Sec Consult releases an advisory regarding multiple vulnerabilities on multiple Zyxel devices"
- start: 2022-12-19
  end: 2023-01-23
  event: "DIVD starts exploring the possibility of EOL devices still being operated, and how to best determine vulnerabilities"
- start: 2023-01-24
  end:
  event: "DIVD scans and notifies owners/oprators of EOL devices"  

> ips: 1

Reviewer: Waarom 1? Case if nog open, dus ips hoeven nu nog niet gezet te worden. 
```suggestion
#ips: 1
```

Author: ivm de comment eronder ging ik er vanuit dat dit een switch was die relevant was als er ips gemoeid waren met de case.

Reviewer: Dit veld wordt gebruikt voor de stats achteraf. Ik zal het in de templates verduidelijken.

Reviewer: Zie #554

# You can set IPs to n/a when this case isn't about IPs (e.g. stolen credentials)
---
## Summary

> On February 15th 2022, [Sec Consult](https://sec-consult.com) published an advisory regarding multiple vulnerabilities on various [Zyxel](https://zyxel.com) devices.

Reviewer: Link naar advisory toevoegen.

> While some of these devices could be patched, others were EOL (end of life) and would not receive a patch. DIVD has performed scans of these devices, to notify owners/operators of the risks regarding the continued operation of these devices.

Reviewer: ```suggestion
While some of these devices could be patched, others were EOL (end of life) and will not receive a patch. DIVD has performed scans for these devices, to notify owners/operators of the risks regarding the continued operation of these devices.
```

## What you can do

> If you currently own or operate any of the listed EOL devices, please consider retiring it as soon as possible. If, for whatever reason, this is not a possibility, make sure all internet facing ports are closed as much as possible.

Reviewer: ```suggestion
If you currently own or operate any of the listed EOL devices, please consider retiring/replacing it as soon as possible. If, for whatever reason, this is not a possibility, make sure all internet facing ports are closed as much as possible.
```

> If you came here through other means than through our direct mailing, please refer to the list published by Sec Consult and Zyxel, to determine whether your Zyel device is vulnerable and if a patch might be available, if your device isn't EOL. 

Reviewer: Link naar de lijst toevoegen.

## What we are doing

> After having notified all owners/operators of relevant EOL Zyxel devices, we will intermittently scan the internet, to determine if appropriate action has been taken, and renotify if we have not received a reply from a devices respective owner/operator.

Reviewer: Je schrijft hier dat je gaat bijhouden wie er reageert en dat je mensen die gaan reageren niet meer gaat mailen. Ga je dat ook echt zo doen? Lijkt mij heel veel en behoorlijk foutgevoelig werk.

Author: Het zijn maar 500 ip adressen. Als er wat binnenkomt en ik dat van de lijst afhaal, lijkt me dat wel te doen. Bij duizenden en duizenden zal het een ander verhaal zijn, zeker.

---
layout: case
title: Multiple Critical Vulnerabilities in multiple Zyxel EOL devices
excerpt: "Based on disclosure by Sec Consult, DIVD performed scans of end of life device impacted by multiple vulnerabilities."
author: Rutger Hermens
lead: Mick Beer
researchers:
- Mick Beer
- Pepijn van der Stap
- Lennaert Oudshoorn
- Rutger Hermens
cves:

> - CVE-2022-4510
> - The researchers described a total of 8 vulnerabilities of which 7 have not been assigned a CVE.

Reviewer: ```suggestion
- n/a
```
